pub/errors/processor.php

```diff
@@ -186,6 +186,15 @@ public function __construct(
         $this->_root  = is_dir($this->_indexDir . 'app');
 
         $this->_prepareConfig();
+
+        if (isset($_ENV['MAGE_RUN_CODE'])) {
```

Comment:
```suggestion
        if (isset($_ENV['MAGE_RUN_CODE'])) {
            $skin = isset($_ENV['MAGE_RUN_TYPE']) ? $ENV_['MAGE_RUN_TYPE'] . '-' . $_ENV['MAGE_RUN_CODE']  : $_ENV['MAGE_RUN_CODE'];
            $this->_setSkin($skin);
        }
```

Reply:
This way you prevent a second request of the same method `_setSkin()`

Reply:
Calling `_setSkin()` multiple times (and in this order) is intentional. The `_setSkin()` method includes some name validation and (importantly) an `is_dir()` check. With the current implementation, a developer can create a per-website theme, even if there is a store code in the web server configuration. With the proposed implementation, a per-website theme is ignored / unusable when a store code is specified in web server configuration.

Reply:
@leonhelmus would it help if I add a code comment to explain why we're calling `_setSkin()` multiple times?